lib/orphans.nix

```diff
@@ -0,0 +1,104 @@
+{ lib }:
+let
+  inherit (lib)
+    types
+    optional
+    isDerivation
+    attrByPath
+    flatten
+    any
+    assertMsg
+  ;
+  inherit (builtins)
+    length
+    concatStringsSep
+    mapAttrs
+    attrValues
+    tryEval
+  ;
+  isAttr = types.attrs.check;
+in
+rec {
+  /* Return a list of maintainers from a package, empty list if inaccessible.
+
+  Example:
+    listMaintainers pkgs.foo
+    => [ { ... } ]
+    listMaintainers pkgs.bar
+    => [ ]
+    listMaintainers {}
+    => error: listMaintainers must receive a package
```

Comment:
Sorry, this is still wrong. Instead of trying to explain (since I am causing more confusion), please just follow this example here: https://github.com/NixOS/nixpkgs/blob/master/lib/attrsets.nix#L15-L23